neqo-transport/benches/min_bandwidth.rs

```diff
@@ -35,52 +37,61 @@ pub fn main() {
 
     const TRANSFER_AMOUNT: usize = GIB;
     const LINK_BANDWIDTH: usize = GBIT;
-    const LINK_RTT_MS: usize = 40;
-    const MINIMUM_EXPECTED_UTILIZATION: f64 = 0.85;
+    const LINK_RTT_MS: u64 = 40;
+    /// The amount of delay that the link buffer will add when full.
+    const BUFFER_LATENCY_MS: usize = 4;
+    /// How much of the theoretical bandwidth we will expect to deliver.
+    const MINIMUM_EXPECTED_UTILIZATION: f64 = 0.75;
 
     let gbit_link = || {
         let rate_byte = LINK_BANDWIDTH / 8;
-        // Router buffer set to bandwidth-delay product.
-        let capacity_byte = rate_byte * LINK_RTT_MS / 1_000;
-        TailDrop::new(rate_byte, capacity_byte, Duration::ZERO)
+        // Set capacity to double when ECN is enabled.
+        let capacity_byte = LINK_BANDWIDTH * BUFFER_LATENCY_MS * (1 + usize::from(ecn)) / 1000;
```

Comment:
```suggestion
        let capacity_byte = LINK_BANDWIDTH * BUFFER_LATENCY_MS / 1000 * (1 + usize::from(ecn));
```

Eases understanding that `/ 1000` converts MS to S.